feature/interface/staticarp/otg_tests/static_arp_test/static_arp_test.go

```diff
@@ -309,20 +279,43 @@ func testFlow(
 	// Get the flow statistics
 	otgutils.LogFlowMetrics(t, otg, config)
 	for _, f := range config.Flows().Items() {
-		recvMetric := gnmi.Get(t, otg, gnmi.OTG().Flow(f.Name()).State())
-		if recvMetric.GetCounters().GetInPkts() != recvMetric.GetCounters().GetOutPkts() || recvMetric.GetCounters().GetInPkts() != 1000 {
+		txPkts := gnmi.Get(t, ate.OTG(), gnmi.OTG().Flow(f.Name()).Counters().OutPkts().State())
+		rxPkts := gnmi.Get(t, ate.OTG(), gnmi.OTG().Flow(f.Name()).Counters().InPkts().State())
+		lossPct := float32(txPkts-rxPkts) * 100 / float32(txPkts)
+		if txPkts == 0 {
+			t.Fatalf("TxPkts == 0, want > 0.")
+		}
+		if lossPct > 0 {
 			t.Errorf("LossPct for flow %s detected, expected 0", f.Name())
+		} else {
+			// Check the egress packets
+			if poisoned {
```

Comment:
From the README + the ATE version test, we should be checking using egress filters both with/without the static (poisoned) entry. The former should do the same check with the configured dest MAC for the ATE dest port.

Reply:
Done